code/datums/diseases/kingstons.dm

```diff
@@ -66,7 +66,7 @@
 	chosentype = pick(virspecies)
 	chosensuff = pick(virsuffix)
 
-	name = "[chosentype] [chosensuff]"
+	name = "[chosentype.name] [chosensuff]"
```

Comment:
I'm pretty sure you'll need to do `initial(chosentype.name)` here, or it'll runtime when you run it.

Reply:
You are correct. Changed now.